internal/cmd/connect/install_utils.go

```diff
@@ -0,0 +1,439 @@
+package connect
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/properties"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/types"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+type platformInstallation struct {
+	Location platformLocation
+	Use      string
+}
+
+type platformLocation struct {
+	Type string
+	Path string
+}
+
+type WorkerConfig struct {
+	Path string
+	Use  string
+}
+
+type listOut struct {
+	Number      string `human:""`
```

Comment:
`Number` is omitted?

Reply:
Yeah I wanted it to show up with an empty header. The number is only there for the interactive prompt which asks the user to pick a number.